site/source/docs/getting_started/downloads.rst

```diff
@@ -1,7 +1,7 @@
 .. _sdk-download-and-install:
 
 ====================
-Download and install
+Download & Install
```

Comment:
This change rather needless.  Why change it?

If you do change it can you change the length of the `====` lines above and below?